README.md

```diff
@@ -1 +1,143 @@
-# Afghnistan-Developers-Conference
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Afghanistan Developers Confrence] <a name="about-project"></a>
+
+> You can find my recent works here. This is also a direct way to contact with me
+
+**[Afghanistan Developers Confrence]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+HTML 
+CSS 
+JAVASCRIPT
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Github Pages</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Home_Page]**
+- **[About_Page]**
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://kazim110.github.io)
+
+## 💻 Loom Video for Introduction<a name="loom-video"></a>
+
+- [Loom Video Link](https://www.loom.com/share/c3f2ab810ae74f3989bd9f2965907a32)
```

Comment:
- [ ] Your video presentation has no audio kindly consider fixing this so the video is clear and presents your pages and some piece of code you are proud of making